inspire_classifier/app.py

```diff
@@ -76,6 +81,35 @@ def page_not_found(e):
         }, 404
 
 
+@click.group(cls=FlaskGroup, create_app=create_app)
+def inspire_classifier():
+    "INSPIRE Classifier commands"
+
+
+@inspire_classifier.command('predict-coreness')
+@click.argument('title', nargs=1)
+@click.argument('abstract', nargs=1)
+def predict(title, abstract):
+    """
+    Usage: inspire-classifier predict-coreness TITLE ABSTRACT
+    """
+    click.echo(predict_coreness(title, abstract))
+
+
+@inspire_classifier.command('train')
+@click.argument('classifier_base_path', type=click.Path(exists=True), required=False, nargs=1)
+def train_classifier(classifier_base_path):
+    """
+    Usage: inspire-classifier train [CLASSIFIER_BASE_PATH]
+    """
+    if classifier_base_path:
+        with app.app_context():
+            app.config['CLASSIFIER_BASE_PATH'] = classifier_base_path
+            train()
```

Comment:
We would want to use [click spinner](https://github.com/click-contrib/click-spinner) here, since the process will take some time. You can also add it to the prediction function.

Reply:
Looks like a nice add-on.

Reply:
I tried this with train and it doesn't exactly help. Since the train method prints out it's own stuff, this interrupts that and occassionally breaks that into new lines, which is undesirable in my opinion. I would say let's keep the spinner for the predict-coreness method only.